docker-compose.dev.yml

```diff
@@ -170,13 +170,11 @@ services:
       - COMPANIES_HOUSE_API_KEY
       - ORDNANCE_SURVEY_API_URL=https://api.os.co.uk
       - ORDNANCE_SURVEY_API_KEY
-      - MULESOFT_API_KEY
-      - MULESOFT_API_SECRET
+      - APIM_TFS_KEY
+      - APIM_TFS_VALUE
```

Comment:
Do we need TFS URL?

Reply:
Yes we do, to keep API micro-service segregated.

Reply:
I couldn't see it in the list 😊

Reply:
Apologies, have added.

Reply:
@abhi-markan  I can't see it in the list?